ecg_segmentation_dataset.py

```diff
@@ -0,0 +1,35 @@
+import torch
+from torch.utils.data import Dataset
+from datasets import Value, Array2D, Features, load_dataset
+
+
+class ECGDataset(Dataset):
+    def __init__(self, huggingface_path: str, split: str = "train"):
+        super().__init__()
+
+        features = Features(
+            {
+                "record_id": Value(dtype="string"),
+                "signal": Array2D(dtype="float32", shape=(2, 1000)),
+                "mask": Array2D(dtype="int8", shape=(1, 1000)),
+            }
+        )
+        self.data = load_dataset(huggingface_path, features=features, split=split)
```

Comment:
```suggestion
        self.data = load_dataset(huggingface_path, split=split)
```

I think it's not necessary to provide features here if we created the dataset including information about datatype? 🤔 Can you check please?

Reply:
I think it's not necessary, but I thought that it would make code a little bit easier to understand if dtypes are specified.

Reply:
If it makes it easier to read for you than it's ok to leave it in. If you ever need to check the datatypes of a dataset you can also use `dataset.features` attribute.